### ISSUE TITLE
Dropdown list for states in new tenant form

### ISSUE THREAD

**Issue by m-henderson** [mentorient/mentorient#47]
How would you all recommend on doing a drop down list for states in the new tenant form? should we create a model for states and then hold all the states in the database?

**Comment by rembrandtreyes** [comment]
I think creating a model would be the proper way to do so. Although since the name of states never change or should changed we can also create a drop down list within HTML.

I vote for a model though, just to keep our HTML cleaner.

**Comment by m-henderson** [comment]
@rembrandtreyes good idea. We can render the states from the database. Also, I apologize for not submitting a lot of PR's this week. I started a new job and have been super busy between that and school! I will catch up this weekend though.
